### PR TITLE
Dockerize transactional email consumer.

### PR DESCRIPTION
**Pull request by sergii-tkachenko**
#### What's this PR do?
- Introduces Docker integration to mb-transactional-email consumer
#### How should this be manually tested?
- See README.md

### CODE REVIEW

**Comment by sergii-tkachenko** [comment]
Note: I removed automatic PHP unit tests. Let's test it manually.

**Comment by DeeZone** [comment]
- Does a port need to be opened to send cURL requests to Mandrill? Will the call to Mandrill work? https://github.com/DoSomething/mbc-transactional-email/blob/master/src/MBC_TransactionalEmail_Consumer.php#L234

- Should the Mandrill test key be used by default? Perhaps a runtime configuration switch for the app?

**Comment by DeeZone** [comment]
I look forward to figuring out how to have useful PHPUnit tests that can be run for each Docker deployment. Do you think this will be difficult? I'm not clear how to manage / test all the dependencies on outside services including the Rabbit server.

Thoughts?

**Comment by sergii-tkachenko** [comment]
@DeeZone 
> Does a port need to be opened to send cURL requests to Mandrill? Will the call to Mandrill work?
No, it just works.

> Should the Mandrill test key be used by default
See Docker configuration: https://github.com/DoSomething/mb-secure-config/pull/1

**Comment by sergii-tkachenko** [comment]
I actually was able to request a new password from Vagrant and received it through dockerized consumer process running on my host.

**Comment by sergii-tkachenko** [comment]
> I look forward to figuring out how to have useful PHPUnit tests that can be run for each Docker deployment. Do you think this will be difficult? I'm not clear how to manage / test all the dependencies on outside services including the Rabbit server.
Thoughts?

Let's discuss ths in the new issue?

**Comment by DeeZone** [comment]
As suggested, new issue: https://github.com/DoSomething/mbc-transactional-email/issues/36

File: README.md [hunk 1]
```diff
@@ -37,3 +37,16 @@ Before deploying to production, don't forget to optimize the autoloader
...
 - `composer install --no-dev`
+
+## Docker
+### Running mbc-transactional-email in Docker container
+First of all, you need to obtain `mb-secure-config`.  
+Let's say, it is saved to `$HOME/Development/mb/mb-secure-config/docker-dev/mb-secure-config.inc`.
+
+
+Then you can start the consumer with:  
+`docker run -v $HOME/Development/mb/mb-secure-config/docker-dev/mb-secure-config.inc:/usr/src/mb/messagebroker-config/mb-secure-config.inc:ro -d --name=mbc-transactional-email dosomething/mbc-transactional-email:latest`
```
Comment (DeeZone) [inline README.md]: This looks specific to your local directory structure. Can we abstract this to work on any user's file structure?

Comment (sergii-tkachenko) [inline README.md]: That's just for example to point out that `mb-secure-config.inc` can be saved anywhere on your local drive.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# mbc-transactional-email
+
+FROM dosomething/mb:latest
+MAINTAINER Sergii Tkachenko <sergii@dosomething.org>
+
+ARG COMPOSER_GITHUB_TOKEN
+ENV RABBITMQ_HOST=192.168.99.10\
```
Comment (DeeZone) [inline Dockerfile]: I don't know enough about Docker so perhaps this is a silly question but will the IP address always be this value?

File: README.md (hunk 1 above)
Comment (DeeZone) [inline README.md]: Ah yes, "Lets say...", a hypothetical situation, got it.

**Comment by DeeZone** [comment]
Will this process involve running `composer install` to setup the application?

File: Dockerfile (hunk 2 above)
Comment (sergii-tkachenko) [inline Dockerfile]: No, you can redefine it. This is just a default value.

**Comment by sergii-tkachenko** [comment]
Yes, I expect `./bin/install.sh` to be executed. The configuration already is in `/usr/src/mb/messagebroker-config/` with `mb-secure-config.inc` mounted in it.

```
root@c09e7ab42f87:/usr/src/mb/mbc-transactional-email# ls -la messagebroker-config
lrwxrwxrwx 1 root root 32 Mar  9 18:10 messagebroker-config -> /usr/src/mb/messagebroker-config
```

**Comment by DeeZone** [comment]
:+1:

**Pull request closed without merging**